puppet/environments/box/modules/profiles/manifests/box/tools.pp

```diff
@@ -35,6 +35,18 @@
     provider => 'npm',
   }
 
+  package { [
+    'libjpeg-dev',
+    'libfontconfig'
+  ]:
+    ensure  => latest
+  }
+
+  exec { 'npm install yellowlabtools' :
+    command => 'npm install -g yellowlabtools --unsafe-perm=true --allow-root',
+    path => ['/usr/bin']
```

Comment:
@yiendos We don't want to run the exec on each Puppet run, so we should add a [creates](https://puppet.com/docs/puppet/latest/type.html#exec-attribute-creates) or [onlyif](https://puppet.com/docs/puppet/latest/type.html#exec-attribute-onlyif) statement. 

`creates` is usually easiest because you can just point to a file (the yellowlabtools executable, for example)